hapi-loader/src/main/java/com/arkhn/hapi/loader/ResourceConsumer.java

```diff
@@ -54,14 +51,9 @@ public ResourceListener resourceListener(MeterRegistry registry) {
         private KafkaProducer producer;
 
         @Autowired
-        private RedisCounterProperties redisCounterProperties;
+        private Jedis redisCounter;
 
-        private Counter failedInsertions;
-        private Counter successfulInsertions;
-
-        ResourceListener(MeterRegistry registry) {
-            failedInsertions = registry.counter("count_failed_insertions");
-            successfulInsertions = registry.counter("count_successful_insertions");
```

Comment:
Why have we removed this? These were grafana counters, right?

Reply:
no idea sorry

Reply:
Hum, we should put them back, what do you think?

Reply:
will it work if resourceListener doesn't inherit from MeterRegistry anymore ?

Reply:
Not sure but we can also put that back, can't we? Maybe I'm missing something but I really don't understand why it's been removed 🤔